lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -178,17 +178,15 @@ fi
 ### Check if repositories are prepared for the upgrade
 # We need to ensure that Cloud5 main repository and update repos are in place
 
-# FIXME repos that do not exist yet
-REPOS_SKIP_CHECKS+=" Cloud SUSE-Cloud-5-Pool SUSE-Cloud-5-Updates"
-
 check_media_content \
     Cloud \
     /srv/tftpboot/suse-11.3/repos/Cloud \
-    1558be86e7354d31e71e7c8c2574031a
+    #1558be86e7354d31e71e7c8c2574031a
```

Comment:
So, we don't need to skip it, but we're not checking md5 sum... isn't it just a diffeernt kind of skip?